### DISCUSSION TITLE
Ability to tag or categorize multiple videos in bulk

### DISCUSSION THREAD

**Discussion by potatowaffle** [mediacms-io/mediacms#588] [Feature Requests]
**Describe the feature you'd like**
I have uploaded over 500 videos to my MediaCMS instance. I need to categorize and tag these in bulk. If there was a way to select multiple videos and categorise them at once it would save hours trying to do this individually.

**Describe alternatives you've considered**
Ability to do this via the CLI tool,and/ or via the Django administration panel

**Additional context**
MediaCMS is great for collections of videos and clips, but we need functionality to do categorize (and tag) multiple clips at once.

**Comment by mgogoulos** [comment]
This should be possible now using the API, so with little help it can be possible with the CLI tool. But indeed it would be much better if it becomes a functionality exposed within the portal. 

It could either be part of manage media (https://portal/manage/media) since there is already an action allowed there (delete media), but this makes it usable only for Admins/Managers/Editors.
Alternatively it could be part of a new interface for managing each authors media (a list view with one's media, and built actions as delete/tag/categorise)
